src/odemis/driver/technolution.py

```diff
@@ -222,14 +220,15 @@ def asmApiGetCall(self, url, expected_status, data=None, raw_response=False, tim
         if resp.status_code != expected_status:
             if isinstance(data, dict):
                 for key, value in data.items():
-                    if isinstance(value, list):
-                        # Convert data elements of type list to string so they can be returned in a logging message
+                    if isinstance(value, list) and len(value) > 10:
+                        # Convert data elements of type list (which have a length bigger than 10) to string so they can
+                        # be returned in a logging message.
                         try:
                             # Limit to first 10 values to not overload error output message
                             data[key] = "First 10 values of the list:" + str(value[0:10])
                         except:
                             data[key] = "Empty - because data cannot be converted to a string"
-                logging.error("Data dictionary used to make call %s contains:\n %s" % (url, str(data)))
+                logging.exception("Data dictionary used to make call %s contains:\n %s" % (url, str(data)))
```

Comment:
Here, it should stay a `.error()`. `.exception()` *only* works in an exception handler block. In the case here, it will show incorrect info.